lib/stat_tracker.rb

```diff
@@ -264,5 +264,62 @@ def lowest_scoring_home
     lowest_scoring=home_calculate_average_goals_per_team.min
     find_team_name(lowest_scoring[0])
   end
-  
+
+  def winningest_coach(season)
+    # Returns a nested hash of information with the coach's name as the outer hash 
+    # The inner has has total games => qty and wins => qty
+    coach_records = {}
+    # Allows you to input the season (which come from the game CSV and only use the 
+    # first four characters which correspond to the year, or season)
+    season_year = season[0..3]
+
+    @game_teams.each do |game|
+      next unless game.game_id[0..3] == season_year
+      
+      coach = game.head_coach
+      result = game.result 
+
+      coach_records[coach] ||= { wins: 0, total_games: 0 }
+      coach_records[coach][:total_games] += 1
+      coach_records[coach][:wins] += 1 if result == "WIN"
```

Comment:
I like how you iterated through the seasons. I think I can probably adapt this for the accuracy methods.